client/src/containers/dashboard/Dashboard.js

```diff
@@ -1,10 +1,14 @@
 import React from "react";
+import { Link } from "react-router-dom";
 
 
 function Dashboard() {
+ 
   return(
     <div data-test="component-dash">
-    <h1>Dashboard</h1>
+    <Link to='/display/images'> Display Images</Link>   
```

Comment:
Tiny note is when you have multiple words in your route, you don't have to slash-seperate them. A `/display-images` would be sufficient. And even a little bit better if SEO (google) would be a thing :)